### PR TITLE
HDDS-8515. Update README to include GitHub Discussions.

### PR DESCRIPTION
**Pull request by kerneltime**
## What changes were proposed in this pull request?

 Update README to include GitHub Discussions.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-8515

## How was this patch tested?

Rendered views 

- https://github.com/apache/ozone/blob/feaab2bdd7a064bac6f8ca9168750e6aac84277d/CONTRIBUTING.md
- https://github.com/apache/ozone/blob/feaab2bdd7a064bac6f8ca9168750e6aac84277d/README.md

### CODE REVIEW

**Comment by kerneltime** [comment]
Rendered view https://github.com/apache/ozone/blob/1d9c9bfb2b107e827da62e7a560b5d4710f951dd/README.md

File: README.md [hunk 1]
```diff
@@ -24,9 +24,11 @@ Ozone is a top level project under the [Apache Software Foundation](https://apac
  * Ozone [web page](https://ozone.apache.org)
  * Mailing lists
      * For any questions use: [dev@ozone.apache.org](https://lists.apache.org/list.html?dev@ozone.apache.org)
- * Chat: You can find the #ozone channel on the official ASF slack. Invite link is [here](http://s.apache.org/slack-invite).
+ * Chat: There are a few ways to interact with the community
+     * You can find the #ozone channel on the official ASF slack. Invite link is [here](http://s.apache.org/slack-invite).
```
Comment (smengcl) [inline README.md:28]:
```suggestion
     * You can find the #ozone channel on the official ASF Slack. Invite link is [here](http://s.apache.org/slack-invite).
```

**Review by smengcl — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -30,6 +30,7 @@ If you have any questions, please don't hesitate to contact
 
  * **email**: use dev@ozone.apache.org.
  * **chat**: You can find the #ozone channel at the ASF slack. Invite link is [here](http://s.apache.org/slack-invite)
+ * **GitHub Discussions**: You can also interact with the community using [GitHub Discussions]https://github.com/apache/ozone/discussions).
```
Comment (adoroszlai) [inline CONTRIBUTING.md:33]:
```suggestion
 * **GitHub Discussions**: You can also interact with the community using [GitHub Discussions](https://github.com/apache/ozone/discussions). 
```

**Review by adoroszlai — CHANGES_REQUESTED** [review]

**Comment by adoroszlai** [comment]
Thanks @kerneltime for the contribution, @smengcl for the review.

**Pull request merged**